lib/fog/vsphere/models/compute/cdrom.rb

```diff
@@ -0,0 +1,83 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Cdrom < Fog::Model
+        attribute :filename
+        attribute :name
+        attribute :key
+        attribute :server_id
+        attribute :controller_key
+        attribute :unit_number
+        attribute :start_connected
+        attribute :allow_guest_control
+        attribute :connected
```

Comment:
unit_number is just unique per controller, we could use key I think.
Are you thinking about ```identity :key```?